aws-inventory/lambda/inventory-firehose.py

```diff
@@ -98,7 +114,9 @@ def discover_firehose(target_account, region):
         resource_item['resourceName']                   = delivery_stream['DeliveryStreamName']
         resource_item['ARN']                            = delivery_stream['DeliveryStreamARN']
         resource_item['errors']                         = {}
+        resource_item['tags']                           = tags
```

Comment:
Will this direct assignment to tags format them in the standard Key/Value method? Boto3 docs tell me that the response is```
     'Tags': [
        {
            'Key': 'string',
            'Value': 'string'
        },
    ]
```
There is a function in common.py called parse_tags() which should be run.

Reply:
added